StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetConfiguration.swift

```diff
@@ -508,6 +511,59 @@ extension PaymentSheet {
         /// - Note: This is always called on the main thread.
         public var externalPaymentMethodConfirmHandler: ExternalPaymentMethodConfirmHandler
     }
+
+    /// Configuration for custom payment methods
+    @_spi(CustomPaymentMethodsBeta) public struct CustomPaymentMethodConfiguration {
+
+        /// Defines a custom payment method type that can be displayed in PaymentSheet
+        public struct CustomPaymentMethod {
+
+            /// The unique identifier for this custom payment method type in the format of "cmpt_..."
+            /// Obtained from the Stripe Dashboard at https://dashboard.stripe.com/settings/custom_payment_methods
+            let id: String
+
+            /// Optional subcopy text to be displayed below the custom payment method's display name.
+            let subcopy: String?
+
```

Comment:
I'm guessing the display name comes from the backend, so I'm curious why the sub copy isn't modeled there as well?  Also, is it awkward that they will control the text, but we control the font/layout, etc. ?

Reply:
Subcopy may need to be dynamic, e.g. a BNPL "Pay in X payments of Y"

Reply:
Assuming this will be a client side parameter, subcopy is feels vague as it has no attachment to the position of  display name -- maybe this is intentional as it gives us freedom to move this text around?  I would have expected the name to be something like `subDisplayNameCopy` or `subDisplayNameText`.

Reply:
Subcopy is currently the same as the web API parameter name, happy to discuss specific naming and things in an API review if you want to be involved.